backend/devops/templates/nginx.conf

```diff
@@ -27,5 +27,14 @@ http {
     #gzip  on;
 
     include /etc/nginx/conf.d/*.conf;
+
+    server {
+      listen 80 default_server;
+
+      location ~ /.well-known {
```

Comment:
This allows certbot to verify that our server is reachable via the domain name for the cert we're creating/renewing. I put this block in `nginx.conf` because `nginx_site.conf` expects the SSL certs to already exist.